--- a/src/zck_gen_zdict.c
+++ b/src/zck_gen_zdict.c
@@ -309,7 +309,7 @@
 #else
     int pid = fork();
     if(pid == 0) {
-        execl("/usr/bin/zstd", "zstd", "--train", dir, "-r", "-o", out_name, NULL);
-        LOG_ERROR("Unable to find /usr/bin/zstd\n");
+        execl("@TERMUX_PREFIX@/bin/zstd", "zstd", "--train", dir, "-r", "-o", out_name, NULL);
+        LOG_ERROR("Unable to find @TERMUX_PREFIX@/bin/zstd\n");
         exit(1);
     }
