diff --git a/misc/fts.c b/misc/fts.c
index bf7fb450a..c19538afb 100644
--- a/misc/fts.c
+++ b/misc/fts.c
@@ -80,7 +80,7 @@ static char sccsid[] = "@(#)fts.c	8.6 (Berkeley) 8/14/94";
 #include <dirent.h>
 #include <errno.h>
 #include "rpmfts.h"
-#   define __set_errno(val) (*__errno_location ()) = (val)
+#   define __set_errno(val) (errno = (val))
 #   define __open	open
 #   define __close	close
 #   define __fchdir	fchdir
