diff --git a/libcomps/src/CMakeLists.txt b/libcomps/src/CMakeLists.txt
index c04a02c..0373acf 100644
--- a/libcomps/src/CMakeLists.txt
+++ b/libcomps/src/CMakeLists.txt
@@ -72,5 +72,8 @@ if (ENABLE_DEVELOPMENT)
           DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig/")
 endif ()
 
-add_subdirectory(python)
+option(ENABLE_PYTHON "Build Python bindings" OFF)
+if(ENABLE_PYTHON)
+    add_subdirectory(python)
+endif()
 
